installer/templates/new/widgets/comments/comments.scss

```diff
@@ -0,0 +1,26 @@
+// ----------------------------------------------------------------------------
+// Sass declarations
+// ----------------------------------------------------------------------------
+$background-color:  #eb9c3c;
+
+$title-color:       rgba(255, 255, 255, 0.7);
+$moreinfo-color:    rgba(255, 255, 255, 0.7);
+
+// ----------------------------------------------------------------------------
+// Widget-Comments styles
+// ----------------------------------------------------------------------------
+.widget-comments {
+  background-color: $background-color;
+
+  .comment-container {
+    h3 {
+      img {
+        border: 1px solid $title-color;
```

Comment:
Properties should be ordered margin-right, border

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)